apps/front/src/components/ui/formButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { CircularProgress } from '@mui/material'
+import ClearIcon from '@mui/icons-material/Clear'
+import { ApolloError } from '@apollo/client'
+
+type FormButtonType = {
+    loading: boolean
+    error?: ApolloError
```

Comment:
ten error nie jest tutaj obsługiwany więc wydaje się że wystarczyłaby flaga `isError`, dzięki temu też nie bedzie wymagania żeby to był koniecznie ApolloError

Reply:
ten Button jest wykorzystywany w formularzach, jeśli przekazuję error jako props to jest on otypowany jako ` MutationResult<TData>.error?: ApolloError | undefined ` Stąd takie otypowanie w FormButtonType

Reply:
no dobra ale na poziomie tego przycisku jest to wymagane aby to był apollo error?